tests/playwright/ui-tests/searchpage.spec.ts

```diff
@@ -13,54 +14,114 @@ test.describe('Search page', () => {
+          await searchPage.goToSearchFor('non')
+          await searchPage.expect.toSeeNoResultsMessage()
+        })
+      })
     })
+  }
+
+  test.describe('Only with JavaScript enabled', () => {
+    test.describe('Given a user searches for a term that returns results', () => {
+      test.beforeEach(async () => {
+        await searchPage.goToSearchFor(searchTerm)
+        await searchPage.expect.toBeOnPageWithResultsFor(searchTerm)
+      })
 
-    test('the user can edit their search and search again', async () => {
-      await searchPage.expect.toSeeSearchInputContainingSearchTerm()
-      await searchPage.searchFor('education')
-      await searchPage.expect.toBeOnPageWithResultsFor('education')
-      await searchPage.expect.toSeeSearchInputContainingSearchTerm()
-      await searchPage.expect.toSeeInformationForEachResult()
+      test('the user can edit their search and select an item using autocomplete', async () => {
+        await searchPage.searchForm.expect.inputToContainSearchTerm()
+        await searchPage.searchForm.typeSearchTerm('education')
+        await searchPage.searchForm.chooseItemFromAutocompleteWithText('Abbey Education')
+        await searchPage.searchForm.submitSearch()
+        await detailsPage.expect.toBeOnTheRightPageFor('Abbey Education')
+      })
     })
 
-    for (const trustResponse of MockTrustsProvider.fakeTrustsResponseData[searchTerm]) {
-      const trustName = trustResponse.GroupName
+    test.describe('Given a user is typing a search term', () => {
+      test.beforeEach(async () => {
+        await searchPage.goTo()
+      })
 
-      test(`when the user clicks on "${trustName}" then it navigates to the details page`, async () => {
-        await searchPage.clickOnSearchResultLinkWithText(trustName)
-        await detailsPage.expect.toBeOnTheRightPageFor(trustName)
+      test('then they should see a list of options and should be able to select one directly', async () => {
+        await searchPage.searchForm.typeSearchTerm(searchTerm)
+        await searchPage.searchForm.expect.toShowAllResultsInAutocomplete()
+        await searchPage.searchForm.chooseItemFromAutocompleteWithText('trust 1')
+        await searchPage.searchForm.submitSearch()
+        await detailsPage.expect.toBeOnTheRightPageFor('trust 1')
       })
-    }
-  })
 
-  test.describe('Given a user searches for a term that returns no results', () => {
-    test('then they can see an input containing the search term so they can edit it', async () => {
-      await searchPage.goToSearchFor('non')
-      await searchPage.expect.toSeeSearchInputContainingSearchTerm()
-    })
+      test('then they should be able to change their search term to a free text search after selecting a result', async () => {
+        await searchPage.searchForm.typeSearchTerm(searchTerm)
+        await searchPage.searchForm.chooseItemFromAutocompleteWithText('trust 1')
+        await searchPage.searchForm.typeSearchTerm('education')
```

Comment:
We should extract usages of `'education'` to a variable (called something like `secondSearchTerm` or `alternativeSearchTerm`) to make it easier to read and replace when we move to a fake database